frontend/src/components/Auth/Login/Login.js

```diff
@@ -6,22 +6,41 @@ import { Formik, Form } from "formik";
 import * as yup from "yup";
 import TextField from "../../TextField/TextField";
 
-const form = {
-  email: "",
-  password: ""
-};
+async function handleSubmit(params) {
+  const FD = new FormData();
+  FD.append("username", params.data.username);
+  FD.append("password", params.data.password);
+
+  fetch("http://127.0.0.1:8000/login", {
```

Comment:
Just a reminder: `ky` will probably be more convenient than `fetch` https://github.com/sindresorhus/ky

Reply:
Thanks! I'll keep it as example to rework using `ky` in near future, when this part of app will be build.